api/groups/networkGroup.go

```diff
@@ -38,9 +40,16 @@ type networkFacadeHandler interface {
 	StatusMetrics() external.StatusMetricsHandler
 	GetAllIssuedESDTs(tokenType string) ([]string, error)
 	GetTokenSupply(token string) (*api.ESDTSupply, error)
+	GetGenesisNodesPubKeys() (map[uint32][][]byte, map[uint32][][]byte, error)
 	IsInterfaceNil() bool
 }
 
+// GenesisNodesConfig defines the eligible and waiting nodes configurations
+type GenesisNodesConfig struct {
+	Eligible map[uint32][][]byte `json:"eligible"`
+	Waiting  map[uint32][][]byte `json:"waiting"`
```

Comment:
please expose them as string (hex representation, aka by using validator pub key converter)

Reply:
why not leaving them as []byte?

Reply:
because in almost every endpoint that we have "access" to return the data in a desired format (vm-query return data excluded) we expose them in human readable format.

Reply:
changed to expose as hex encoded string